src/main/play-25/uk/gov/hmrc/play/config/AccessibilityStatementConfig.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.play.config
+
+import java.net.URLEncoder
+import javax.inject.Inject
+import play.api.Configuration
+import play.api.mvc.RequestHeader
+
+class AccessibilityStatementConfig @Inject()(config: Configuration) {
+  val platformHost: Option[String] = config.getString("platform.frontend.host")
+  val accessibilityStatementHost: Option[String] =
+    platformHost.orElse(config.getString("accessibility-statement.host"))
```

Comment:
We could make this a little shorter if we're worried about length:
```
class AccessibilityStatementConfig @Inject()(config: Configuration) {
  val accessibilityStatementHost: Option[String] =
    config.getString("platform.frontend.host")
         .orElse(config.getString("accessibility-statement.host"))
```

Reply:
platformHost is used further down though.